ui/prometheus-plugin/src/plugins/prometheus-datasource.ts

```diff
@@ -36,10 +38,10 @@ const createClient: DatasourcePlugin<PrometheusDatasourceSpec, PrometheusClient>
     options: {
       datasourceUrl,
     },
-    instantQuery: (params) => instantQuery(params, { datasourceUrl }),
-    rangeQuery: (params) => rangeQuery(params, { datasourceUrl }),
-    labelNames: (params) => labelNames(params, { datasourceUrl }),
-    labelValues: (params) => labelValues(params, { datasourceUrl }),
+    instantQuery: (params) => instantQuery(params, { datasourceUrl, headers }),
```

Comment:
Small nit: At some point, it might make sense to pass this as an object to all of the query methods.
```tsx
const queryOptions = { datasourceUrl, headers };
...
instantQuery: (params) => instantQuery(params, queryOptions),
```